lib/LaTeXML/Package/LaTeX.pool.ltxml

```diff
@@ -140,9 +140,9 @@ Let('\@currenvline', '\@empty');
 
 DefMacro('\begin{}', sub {
     my ($gullet, $env) = @_;
-    my $name   = $env && ToString($env);
-    my $before = LookupValue('@environment@' . ToString($_[1]) . '@beforebegin');
-    my $after  = LookupValue('@environment@' . ToString($_[1]) . '@atbegin');
+    my $name   = $env && ToString(Expand($env));
```

Comment:
question: Shouldn't this be realized via the parameter type? E.g. replacing the `{}` via an `XToken` or such? cc @brucemiller

Reply:
Also, if `$name` is undefined / empty we probably want to throw an error

Reply:
Well, yeah, sorta, but ... No. There's  an Expanded parameter type which we tried to use. It failed because it kinda expands too early(?). Well, really LaTeX gets the argument which might be balanced braces or a single token, eventually that gets expanded.  Using Expanded caused latexml to only see the 1st char of the expansion instead of the whole thing; adding extra braces mucked things up differently.  It's probably true that one would need (at least) two separate flavors of "expanded"...

Reply:
Right, it looks like we need a `->readXArg` rather than a `->readXToken`, and since we don't have that yet, the current PR makes sense

Reply:
But this begs the question if the `Expanded` parameter type is ever the correct type to use? Maybe it just needs an upgrade, even separately from the `\begin` case.